source/includes/administration/_environments.md

```diff
@@ -244,10 +243,12 @@ You will need the `Environments update` permission to execute this operation.
 
 `DELETE /environments/:id`
 
+Purge an environment and its underlying resources asynchronously. If such purge operation fail for the first time, the service account of this environment would be updated to `ERROR_PURGING`. It would force purge an environment an `ERROR_PURGING` sercice account.
```

Comment:
"it would force purge an environment and `error_purging` service account"
service account typo ! 
This seems a bit unclear to me. I think it should probably say something like: 
if called on an environment in error purging state then it would force purge the service account.

Reply:
updated